<launch>


<!-- car controller -->
<node name="waypoint_manager" pkg="waypoint_manager" output="screen" type="waypoint_manager_node">
<rosparam
    command="load" file="$(find waypoint_manager)/config/pid_param.yaml" />
<param name="waypoints_file" value="$(find waypoint_manager)/config/track.txt"/>
<remap from="cmd_vel" to="catvehicle/cmd_vel" />
</node>

</launch>
